app/controllers/jobseekers/unlocks_controller.rb

```diff
@@ -1,3 +1 @@
-class Jobseekers::UnlocksController < Devise::UnlocksController
-  after_action :replace_devise_notice_flash_with_success!, only: %i[show]
-end
+class Jobseekers::UnlocksController < Devise::UnlocksController; end
```

Comment:
We can get rid of this entirely 🚮

Reply:
I thought we could, but it resulted in some tests failing with "can't find the unlocks controller"... I guess because we were using `jobseeker_unlocks_path`, created by routes:

https://github.com/DFE-Digital/teaching-vacancies/blob/6392539ed212e593b3b482fc1ccca18d792b73e4/config/routes.rb#L18

I'll investigate what we can change here.

Reply:
Ah. You should be able to remove the custom route, and then it'll pick up the default Devise one!

Reply:
It seems that we need to keep our custom controller in order to use our custom view.